src/main/java/com/google/sps/data/Profile.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+import java.util.ArrayList;
+
+/**
+ * Class representing user profile
+ */
+public class Profile {
+
+  private long id;
+  private String userName;
+  private String userEmail;
+
+  // dietary options
+  private boolean vegetarian;
+  private boolean vegan;
+  private boolean glutenFree;
+  private boolean dairyFree;
+
+  private String[] allergies;
```

Comment:
Do you plan to allow all raw text values?

Reply:
Yes, but later on there might be options for the user to choose from so that random values won't get used.